src/askui/telemetry/pkg_version.py

```diff
@@ -0,0 +1,14 @@
+from importlib.metadata import PackageNotFoundError, version
+
+from askui.logger import logger
+
+
+PKG_NAME = "askui"
```

Comment:
Why do we need to define the package name here? Can we not retrive this dynamically?

Reply:
I decided to do it this was as it was easier and I assumed that it changes rarely if ever. Nevertheless, I think this could be a source of issues if we forget so I switched to dynamically determining this.